vscode/package.json

```diff
@@ -68,6 +68,21 @@
         "title": "Run Analysis",
         "category": "Konveyor",
         "icon": "$(play}"
+      },
+      {
+        "command": "konveyor.loadStaticResults",
+        "title": "Load results from files",
+        "category": "Konveyor"
+      },
+      {
+        "command": "konveyor.cleanRuleSets",
+        "title": "Reset analysis results",
+        "category": "Konveyor"
+      },
+      {
+        "command": "konveyor.loadRuleSets",
+        "title": "Load analysis results",
+        "category": "Konveyor"
```

Comment:
I'm curious if these are something we only see being useful for our development workflows or useful to end users as well. My suspicion is that it best serves us in development.

We do not have to include commands in this file in order to call them like `vscode.commands.executeCommand("konveyor.loadRuleSets", rulesets);`, only register them ([ref](https://stackoverflow.com/a/55271392)).

For example, maybe we omit them from here and have a "development" panel in our webview to access some of these commands that we need for development purposes.

Reply:
Fair enough. I think we should maybe consider using the when clause to limit when some of these palette commands are available.

https://code.visualstudio.com/api/references/when-clause-contexts

We should just create an issue so we can prioritize this.

Reply:
IMHO it's low priority. Adding `when` condition will basically limit the command visibility to some context i.e. user will need to open our panels to see the command. Once we reach maturity it may be an elegant way of saying that the command is internal/debug/dev etc.